hedera-node/src/main/java/com/hedera/services/state/submerkle/ExpirableTxnRecord.java

```diff
@@ -390,52 +347,8 @@ public void deserialize(SerializableDataInputStream in, int version) throws IOEx
 		}
 		// Added in 0.21
 		alias = ByteString.copyFrom(in.readByteArray(Integer.MAX_VALUE));
-		// Added in 0.23
-		deserializeAllowanceMaps(in, version);
 	}
 
-	private void deserializeAllowanceMaps(SerializableDataInputStream in, final int version) throws IOException {
```

Comment:
This can't be simply dropped. It will mess up loading v0.23 and v0.24 data. You need to read the bytes still, despite not using them.

Reply:
Thanks Ray ! Agreed. Opened https://github.com/hashgraph/hedera-services/pull/3183.